src/index.js

```diff
@@ -1,17 +1,37 @@
-let Promise = require('bluebird')
+let debug = require('debug')('contexture:mongo')
+
+let revolvingCounter = max => {
+  let counter = 0
+  return {
+    inc() {
+      if (counter === max) {
+        counter = 1
+      } else {
+        counter++
+      }
+      return counter
+    },
+  }
+}
+let counter = revolvingCounter(500)
```

Comment:
I'm not sure if we need the above. The thought behind it was that large numbers are not super helpful when debugging.